src/chrome/content/rules/Tampermonkey.xml

```diff
@@ -0,0 +1,30 @@
+<!--
+	For Tampermonkey.net
+
+		Works:
+
+			- www
+			- $
+			- forum
+			- safari
+
+		Doesn't Work:
+		nil
+-->
+<ruleset name="Tampermonkey.net">
+  <target host="tampermonkey.net" />
+  <target host="*.tampermonkey.net" />
+
+  <rule from="^http://tampermonkey\.net/"
+          to="https://tampermonkey.net/" />
+
+  <rule from="^http://(www|forum|safari)\.tampermonkey\.net/"
```

Comment:
Style guide is live: https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md. Sorry for all the churn in how we write rules. I think in the end it will make for a much more maintainable ruleset library!